mlir/test/Integration/Dialect/SparseTensor/CPU/sparse_sum_bf16.mlir

```diff
@@ -29,7 +30,7 @@
 // Do the same run, but now with  VLA vectorization.
 // RUN: %if mlir_arm_sve_tests %{ %{compile_sve} | %{run_sve} | FileCheck %s %}
 
-// UNSUPPORTED: target=aarch64{{.*}}
+// UNSUPPORTED: mlir_arm_emulator
```

Comment:
I see now that these tests generally don't compile for AArch64 (unrelated to emulation). So, I think this should be something like `UNSUPPORTED: mlir_arm_emulator, target=aarch64{{.*}}`. 

i.e. supported under emulation and real hardware.

Reply:
ok, I added back target=aarch64{{.*}}